extensions/attributegen/testdata/server.yaml

```diff
@@ -0,0 +1,236 @@
+node:
+  id: test-server-ratings
```

Comment:
Please use the test framework to add an integration test. I don't really trust the mocking framework. That would help save on duplication of the JSONs in this repo, since you can use one of the templates.